Test/Unit/Block/JsTest.php

```diff
@@ -241,4 +247,55 @@ public function setSandboxMode($value = true)
             ->method('isSandboxModeSet')
             ->will($this->returnValue($value));
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetJavascriptSuccess()
```

Comment:
You aren't testing anything meaningful here :)
This test is testing mock framework, not our code.
I suggest removing this.

Reply:
removed